.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 🐛 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  Issues are so 🔥
+
+  If you remove or skip this template, you'll make the 🐼 sad and the mighty god
+  of Github will appear and pile-drive the close button from a great height
+  while making animal noises.
+
+  👉🏽 Need support, advice, or help? Don't open an issue!
+  Head to StackOverflow or https://gitter.im/webpack/webpack.
+-->
+
+* Operating System:
+* Node Version:
+* NPM Version:
+* webpack Version:
+* webpack-defaults Version:
```

Comment:
It would be better to give a simple command to run. That's what they do with Jest. Likely you get `npx <something>` and then they'll paste the output.

Reply:
I'm not sure I understand the context here. `webpack-defaults` doesn't have a CLI, so there's no command to run.

Reply:
Ah, I missed the context. It's probably a separate PR but the point was to do that for projects using defaults so people can generate the info easier. That said, such tool would likely be useful here as well.